.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,17 @@
+name: deploy
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  main:
+    if: ${{ github.event_name == 'push' }}
```

Comment:
Note: this is technically not needed as we only specify the `push` trigger for the `on` value.

Reply:
Oop, this is leftover from earlier experimenting. Removing.